Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -105,6 +105,7 @@
 workflows[140.51] = ['',['RunHI2010','REPACKHID','RECOHID11St3','HARVESTDHI']]
 workflows[140.52] = ['',['RunHI2010','RECOHID10','RECOHIR10D11','HARVESTDHI']]
 workflows[140.53] = ['',['RunHI2011','RECOHID11','HARVESTDHI']]
+workflows[140.54] = ['',['RunPA2013','RECOUP15_PPbData','HARVESTUP15_PPbData']]
```

Comment:
The data input is 2013 (Run1) while the era setup is RECOUP15 (which points to Run2 era customizations).
This is asking for trouble and will probably crash in CSC.